### ISSUE TITLE
Connectors page bottom paragraph has insufficient color contrast ratio.

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#82329]
### Description

Paragraphs in the Connectors page use a default color that produces insufficient color contrast ratio.

### Step-by-step reproduction instructions

- Go to Settings > Connectors.
- Observ ethe paragraph at the bottom of the page with text `If the connector you need is not listed, [search the plugin directory](http://localhost:8889/wp-admin/plugin-install.php?s=connector&tab=search&type=tag) to see if a connector is available.`
- Observe the paragraph color is `#949494`.
- Observe `#949494` produces insufficient color contrast ratio against the white background.
- Observe the related CSS ruleset applies to all paragraphs in that page:

```
.connectors-page>p {
    color: #949494;
}
```

The color `#949494` should not be used for text.
Similar paragraph in the Fonts page use `#757575` 

### Screenshots, screen recording, code snippet

<img width="1571" height="674" alt="Image" src="https://github.com/user-attachments/assets/296181d2-73de-4074-a634-e46226f67bb3" />

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [x] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by jasmussen** [comment]
This does not need design feedback, it's a bug that needs a patch, which I would encourage anyone to make.

**Comment by yyppsk** [comment]
I'd like to take this one.
